modules/op-starting-pipelines.adoc

```diff
@@ -0,0 +1,48 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc
+
+[id="op-starting-pipelines_{context}"]
+= Starting Pipelines
+
+After you create a Pipeline, you need to start it for the Pipeline to run and execute its Tasks. You can start a Pipeline Run from the *Pipelines* view, *Pipeline Details* page, or the *Topology* view.
+
+.Procedure
+To start a Pipeline using the *Pipelines* view:
+
+. In the *Pipelines* view of the *Developer* perspective, click the *Options* {Kebab} menu adjoining a Pipeline, and select *Start*.
+. The *Start Pipeline* dialog box displays the *Git Resources* and the *Image Resources* based on the Pipeline definition.
++
+[NOTE]
+====
+For Pipelines created using the *From Git* option, the *Start Pipeline* dialog box displays an `APP_NAME` field in the *Parameters* section, in addition. All the fields in the dialog box are prepopulated by the Pipeline templates in the *From Git* flow.
```

Comment:
s/the *Start Pipeline* dialog box displays an `APP_NAME` field in the *Parameters* section, in addition./ the *Start Pipeline* dialog box displays an additional `APP_NAME` field in the *Parameters* section.